### DISCUSSION TITLE
[SOS!!!] 生产环境milvus异常挂了

### DISCUSSION THREAD

**Discussion by RongHouchao** [milvus-io/milvus#22625] [Q&A and General discussion]
日志信息：
![image](https://user-images.githubusercontent.com/117245915/223636009-159e18a9-c374-4e83-a889-8a119e5fbce3.png)

**Comment by xiaofan-luan** [comment]
Milvus的版本是多少，感觉新版本不应该有类似问题了？挂掉以后拉起无法恢复么？如果无法恢复可能需要提供完整的日志。
看起来流里的数据有乱序，如果不能恢复的话可能要跳掉一部分数据了
